app/components/ak-text-field/index.ts

```diff
@@ -11,6 +11,7 @@ export interface AkTextFieldSignature {
   Args: {
     formControlClass?: string;
     textFieldRootClass?: string;
+    textInputRootClass?: string;
```

Comment:
we can remove this

Reply:
My styles were not getting applied if was not adding this